package.nls.json

```diff
@@ -421,15 +421,19 @@
     "AWS.s3.downloadFile.progressTitle": "Downloading {0}...",
     "AWS.s3.downloadFile.saveButton": "Download",
     "AWS.s3.downloadFile.error.general": "Failed to download file {0}",
+    "AWS.s3.fileViewer.button.dismiss": "Don't show this again",
+    "AWs.s3.fileViewer.button.help": "Help",
     "AWS.s3.fileViewer.message.tempCreation": "Temp folder for FileViewer created with location: {0}",
-    "AWS.s3.fileViewer.warning.noSize": "File size couldn't be determined. Continue with download?",
-    "AWS.s3.fileViewer.warning.4mb": "File size is more than 4MB. Continue with download?",
     "AWS.s3.fileViewer.message.fileKey": "Retrieving and displaying file {0}",
     "AWS.s3.fileViewer.message.fileLocation": "File to be opened is: {0}",
     "AWS.s3.fileViewer.message.sizeLimitCancellation": "Download cancelled",
     "AWS.s3.fileViewer.message.noSizeCancellation": "Download cancelled",
     "AWS.s3.fileViewer.error.download": "Error downloading file {0} from S3: {1}",
-    "AWS.s3.fileViewer.error.invalidSize": "Files over 50MB currently not supported for file display, please use the \"download as\" function",
+    "AWS.s3.fileViewer.error.invalidSize": "Files over 50MB currently not supported for file display, please use the \"Download as...\" function",
+    "AWS.s3.fileViewer.error.invalidUpload": "File is invalid to upload, file has changed in S3 since last cache download. Please try and reopen the file. Be aware current changes may be lost.",
```

Comment:
Changed message to "File is invalid to upload, file has changed in S3 since last cache download. Please compare your version with the one in S3. Then decide if you want to overwrite them or cancel this upload."
well if they cancel upload, it will just return after selection, and keep those changes, else it will continue with the download